pints/toy/_stochastic_logistic_model.py

```diff
@@ -0,0 +1,160 @@
+#
+# Stochastic degradation toy model.
```

Comment:
can we change to stochastic degradation model?

Reply:
and make the name consistent throughout!

Reply:
Done.